app/logic/emailHandler.py

```diff
@@ -97,6 +100,11 @@ def retrieve_recipients(self, recipients_category):
                 .join(EventRsvp)
                 .where(EventRsvp.event==self.event.id))
 
+        if recipients_category == "Eligible Students":
+            bannedUsers = ProgramBan.select(ProgramBan.user_id).where((ProgramBan.endDate > datetime.now()) | (ProgramBan.endDate is None), ProgramBan.program_id.in_([p.id for p in self.program_ids]))
+            allVolunteer = Event.select().where(Event.isAllVolunteerTraining == True)

```

Comment:
We need to make sure we get the all volunteer event for this particular academic year